src/api/search/package.json

```diff
@@ -21,6 +24,9 @@
     "node": ">=12.0.0"
   },
   "devDependencies": {
-    "nodemon": "2.0.15"
+    "@senecacdot/eslint-config-telescope": "1.1.0",
```

Comment:
We need to delete supertest.

Reply:
The `Eslint` is complaining that we need to put `supertest` in the devDependencies.

Reply:
I think whatever from `jest` we use like `nock`, `jest-fetch-mock`, we can add to devDependencies. Eslint complained to me about it too

Reply:
so eslint does not like it when the `supertest` and `nock` are not listed in package.json. We can either add it to the `devDependencies` or add a ignore line right on top of it when we request it.

Reply:
Personally, I think it's ok to add it to package.json